README.md

```diff
@@ -72,8 +84,12 @@ class MyDaemonizedCommand extends DaemonCommand
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $output->writeln("Iteration");
-
-        usleep(100000);
+        
+        // This method helps to give back the CPU to the react-loop (here for 1000 $µseconds).
+        // So you can wait between two iterations if your workers has nothing to do.
+        
+        // We do not recommand to use sleep() or usleep() anymore.
+        $this->setNextIterationSleepingTime(1000); // Every milliseconds
```

Comment:
you should use another value, this one may be confusing because you put `1000` and say `milliseconds`

I'd suggest to either explicitly say `Every 1000 microseconds`

or to use another value instead:

```php
$this->setNextIterationSleepingTime(1000000); // Every second
```